src/js/big-play-button.js

```diff
@@ -34,7 +34,22 @@ class BigPlayButton extends Button {
    * @listens click
    */
   handleClick(event) {
-    this.player_.play();
+    const playPromise = this.player_.play();
+
+    const cb = this.player_.getChild('controlBar');
+    const playToggle = cb && cb.getChild('playToggle');
+
+    if (!playToggle) {
+      return;
```

Comment:
Makes sense, I'll just focus the player element and we can always tweak it in the future if we think there's something better.